pkgs/applications/networking/instant-messengers/teams-for-linux/default.nix

```
@@ -13,17 +13,19 @@
 , pipewire
 , alsa-utils
 , which
+, testers
+, teams-for-linux
 }:
 
-stdenv.mkDerivation (finalAttrs: {
+stdenv.mkDerivation (finalAttrs: rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

Can you still take out this `rec`? Otherwise this looks great now, thank you!